### DISCUSSION TITLE
Horizontal notation for Euclidean vectors

### DISCUSSION THREAD

**Discussion by StevenClontz** [TeamBasedInquiryLearning/linear-algebra#312] [General]
It'd be handy to formally introduce notation for Euclidean vectors that's horizontal, e.g. when communicating in LMS/emails where typesetting the column vector is pain.

We probably should avoid $(a,b,c)$ due to the use of ordered pairs in AT5.

Maybe $[a;b;c]$? That matches with how Octave interprets `[a;b;c]`:

![image](https://github.com/TeamBasedInquiryLearning/linear-algebra/assets/1559632/300208d0-ce36-4716-9c36-715126012ebd)

**Comment by StevenClontz** [comment]
We could more generally allow semicolons as line breaks in any matrix notation, so `\left[\begin{array}{ccc}a & b & c \\ d & e & f\end{array}\right]`$=[a\hspace{0.2em}b\hspace{0.2em}c;d\hspace{0.2em}e\hspace{0.2em}f]$...

**Comment by siwelwerd** [comment]
I think this introduces a confusion that we were very intentional to avoid by *always* using columns for vectors.  But if we did do such a thing, we should not invent a new notation, but rather use the transpose as is standard.  

I would advocate very strongly for never using this in the book at all, other than to add an aside/remark in the first section noting something like "When vertical space in typesettting is an issue, many people express this with $[a\ b\ c]^T$ ...".  Even if the instructor never wants to use the notation, they are likely to encounter it in other textbooks in other classes.

**Reply by StevenClontz** [reply]
On this note, it's interesting we don't introduce transpose at all. We don't need it, but it's one of those things someone might just assume was seen in this class. We could always add it to the determinant results.

**Reply by siwelwerd** [reply]
In my mind, that omission is also very intentional, as a transpose is a thing you do to a matrix, which is a linear transformation, and what exactly does it mean to transpose a linear transformation?  I don't know how to answer that without discussing dual spaces or inner product spaces.

**Reply by StevenClontz** [reply]
Yeah. I pulled a couple textbooks off the shelf, and I don't see any clever motivation there. So using the T superscript as a formal notation for "this should be stacked as a Tower" or something seems fine to me.

**Comment by StevenClontz** [comment]
#314 implements this, so closing discussion
